python/src/opendp/context.py

```diff
@@ -372,7 +423,7 @@ def __init__(
     ) -> None:
         """Initializes the query with the given chain and output measure.
 
-        It is more convenient to use the `context.query()` constructor than this one.
+        It is more convenient to use the ``context.query()`` constructor than this one.
```

Comment:
Make this a pyfunc to user can link to it

Reply:
fixed locally.